packages/core/src/external/carequality/ihe-gateway-v2/ihe-gateway-v2-logic.ts

```diff
@@ -116,16 +117,14 @@ export async function createSignSendProcessXCPDRequest({
   });
   for (const result of results) {
     try {
+      // TODO not sure if we should retry on timeout
```

Comment:
@jonahkaye can you help checking if we can/should retry on timeouts for this and the other `TODO not sure...`? They're all related to IHE GW v2.

Reply:
so xcpd is indempotent and we can do retries on timeouts there and that will get rid of all the annoying PD errors during deployments. 
DQs and DRs arent though unless we add something to deduplicate them at the beggining of `processOutboundDocumentQueryResps` and `processOutboundDocumentRetrievalResps`

Reply:
As discussed on slack, this issue is kinda complicated and solving it is beyond the scope of this PR.